services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -17,21 +17,35 @@ export const ExportedModalDrawerReportSection = ({
   const sectionHeading = verbiage?.intro.subsection || name;
   const entityCount = report?.fieldData?.[entityType]?.length;
 
-  let emptyEntityMessage = "";
+  let emptyEntityMessage;
   switch (entityType) {
     case ModalDrawerEntityTypes.ACCESS_MEASURES:
-      emptyEntityMessage = `No access and adequacy measures have been entered for this program report.`;
+      emptyEntityMessage = " 0 -- No access measures entered";
```

Comment:
Thought: Instead of including the number in the empty entity message, you could change line 18 to return either the current code or 0, like: 

```
const entityCount = report?.fieldData?.[entityType]?.length || 0;
```

Then `emptyEntityMessage` is just `No access measures entered` or whichever is right. Then we can simplify below (see my note there).